partner_contact_nutrition/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Contact Nutrition Information",
+    "summary": "Provide caloric intake",
+    "version": "10.0.1.0.1",
```

Comment:
@max3903 attended in https://github.com/OCA/partner-contact/pull/349/commits/e6b7fc4c2214038bef8bc969ac28f59047f6f481